lib/mix/tasks/compile.thrift.ex

```diff
@@ -133,21 +133,29 @@ defmodule Mix.Tasks.Compile.Thrift do
     end
   end
 
+  defp thrift_vsn do
```

Comment:
This name makes me think it's the actual name of the version of thrift supported (e.g., 0.9.3), which kind of opens up a can of worms... so far we seem to be 0.9.3 compatible and thrift seems to be backwards compatible, but does anyone know if they have made any statements about potential for backwards incompatibility?  If thrift 1.0.0 was not backwards compatible with 0.9.x, how would we handle that?